deploy/grafana/dashboards/clusters.json

```diff
@@ -107,7 +107,7 @@
     {
       "datasource": {
         "type": "prometheus",
-        "uid": "${datasource}"
+        "uid": "${DS_PROMETHEUS}"
```

Comment:
> Update datasource variable to DS_PROMETHEUS convention

Do you have a link to this convention? I just glanced at the [top dashboard on Grafana.com](https://grafana.com/grafana/dashboards/1860-node-exporter-full/), and it uses `datasource` rather than `DS_PROMETHEUS` as the variable. Seems the same for the built-in dashboards in Azure's managed Grafana.

I'm also afraid that this might be a breaking change to someone's existing dashboard setup.

Reply:
I've observed this "naming convention" widely used in recent years. If the dashboard needs to incorporate another type of datasource in the future, the name will clearly indicate the type.

Reply:
I can revert the change if you prefer.

Reply:
I see. Added some thoughts here https://github.com/microsoft/retina/pull/158#pullrequestreview-2226013402